docs/spelling_wordlist.txt

```diff
@@ -296,6 +296,7 @@ ContainerPort
 contentUrl
 contextmgr
 contrib
+CoreV
```

Comment:
This is a little suspicious...

Reply:
YEah.  I tried different solutions in attemps 70-99 or so.

I have no better way to solve it :( . Not even sure why `CoreV1Api` gets into spellchecker (It is missing in the generated .rst document). Lilly bug in autoapi which I think is not worth investigatig. CoreV in spalling wordlist solves it and that was about